nixos/roles/kvm.nix

```diff
@@ -4,16 +4,23 @@ with builtins;
 
 let
   fclib = config.fclib;
-  role = config.flyingcircus.roles.kvm_host;
+  cfg = config.flyingcircus.roles.kvm_host;
```

Comment:
That's mainly a matter of taste, but okay, I am not going to die on that hill.
The distinction between `role` and `service` (latter often also `cfg`) is useful in cases where we have a role configuring a corresponding service module in the same file, so that the distinction between those 2 scopes is immediately clear. This is not strictly the case here, so feel free to change it like this.
Just some background on why the naming was like this and how I would not support generally changing this throughout the whole platform code.